packages/sessions-sdk-react/src/session-provider.tsx

```diff
@@ -214,6 +214,20 @@ const SessionProvider = ({
           />
         )}
       </ModalDialog>
+      <ModalDialog
+        heading="This trade exceeds your set limits"
+        message="Would you like to increase your session limits?"
+        isOpen={sessionState.type === StateType.RequestingIncreasedLimits}
+        onOpenChange={onExtendSessionOpenChange}
+      >
+        {isEstablished(sessionState) && (
+          <RenewSessionsContents
+            sessionState={sessionState}
+            enableUnlimited={enableUnlimited}
```

Comment:
if the trade exceeds set limits, then should this be `false`? as the user has already created a limited session?

Reply:
no this option means the app has requested unlimited sessions to be an available option to the user, not that the user has selected it